grow/testing/testdata/extract_localized/unlocalized_pod/content/unlocalized_collection/unlocalized_doc_with_localized_parts.yaml

```diff
@@ -1,5 +1,6 @@
----
+#---
 foo@: Unlocalized base doc part in unlocalized collection in unlocalized pöd
----
-$locale: ja
-foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
+#---
+## Since $locales: ja is not specified anywhere, this fails to extract.
+#$locale: ja
+#foo@: Localized doc part in unlocalized doc in unlocalized collection in unlocalized pöd
```

Comment:
Might as well just remove this?

Don’t think we need to test that a commented out string isn’t extracted (it doesn’t appear anywhere else).